package.json

```diff
@@ -29,6 +29,7 @@
     "angular-gettext": "2.3.4",
     "angular-gettext-tools": "2.2.3",
     "angular-messages": "1.5.8",
+    "slug": "0.9.1",
```

Comment:
Depending on the npm version `slug` might be installed in `node_modules/slug` or in `node_modules/angular-slug/node_modules/slug`. When using an explicit dependency, it will always be in `node_modules/slug`.